src/util/string_util.c

```diff
@@ -800,3 +800,37 @@ double string_to_double(const char *str, ErrorStack *error_stack) {
   }
   return result;
 }
+
+char *json_unescape_string(const char *json_string, ErrorStack *error_stack) {
+  if (!json_string) {
+    error_stack_push(error_stack, ERROR_STATUS_CONFIG_LOAD_MISSING_ARG,
+                     string_duplicate("json_string cannot be null"));
+    return NULL;
+  }
+  
+  int input_len = strlen(json_string);
```

Comment:
You can remove the error check and just use string_length(json_string), which will log_fatal on NULL, and so I guess you can remove error_stack from the signature at that point.